modules/ml/src/main/java/org/apache/ignite/ml/inference/IgniteModelStorageUtil.java

```diff
@@ -112,7 +135,20 @@ private static void saveModelStorage(byte[] serializedMdl, UUID mdlId) {
 
         ModelStorage storage = new ModelStorageFactory().getModelStorage(ignite);
         storage.mkdirs(IGNITE_MDL_FOLDER);
-        storage.putFile(IGNITE_MDL_FOLDER + "/" + mdlId, serializedMdl);
+        storage.putFile(IGNITE_MDL_FOLDER + "/" + mdlId, serializedMdl, true);
+    }
+
+    /**
+     * Removes model with specified identifier from model storage.
+     *
+     * @param mdlId Model identifier.
+     */
+    private static void removeModelStorage(UUID mdlId) {
+        Ignite ignite = Ignition.ignite();
+
+        ModelStorage storage = new ModelStorageFactory().getModelStorage(ignite);
+        storage.mkdirs(IGNITE_MDL_FOLDER);
```

Comment:
there is no need to "mkdirs" here
you have added directories after first save

Reply:
Removed.